tempesta_fw/http_parser.c

```diff
@@ -563,89 +582,68 @@ do {									\
 
 #define TFW_HTTP_PARSE_BODY(prefix)					\
```

Comment:
No sure that we can do it in a such manner as `RGen__EoL` state now needs to jump to `RGen_Body` state which will impossible to address if it will resides in a separate function. Or may be I don't understand of what you meant?